### PR TITLE
refine graph doc

### PR DESCRIPTION
**Pull request by Ldpe2G**
代码改动部分，对应的截图
image
image

### CODE REVIEW

File: python/oneflow/nn/graph/graph.py [hunk 1]
```diff
@@ -150,15 +150,17 @@ def build(self, *args, **kwargs):
...
             ...         super().__init__()
-            ...         self.linear = flow.nn.Linear(3, 8, False)
+            ...         self.model = linear
             ...     def build(self, x):
-            ...         return self.linear(x)
+            ...         return self.model(x)
 
             >>> linear_graph = MyGraph()
             >>> x = flow.randn(4, 3)
+            >>> model.eval() # make linear module executing in evaluation mode
```
Comment (strint) [inline python/oneflow/nn/graph/graph.py:163]: linear.eval() ？ doc test应该过不了？

File: python/oneflow/nn/graph/graph.py [hunk 2]
```diff
@@ -217,8 +219,8 @@ def add_optimizer(
         * learn rate scheduler's ``step()``.
 
         Also note that only scalar tensor are allowed to call ``backward()``
-        in ``nn.Graph.build()`` for the moment. So you may call ``Tensor.sum()``
-        or ``Tensor.mean()`` to make the loss tensor a scalar tensor.
+        in ``nn.Graph.build()`` for the moment. So you may call
+        ``Tensor.mean()`` to make the loss tensor a scalar tensor.
```
Comment (strint) [inline python/oneflow/nn/graph/graph.py:223]: 这里为啥完全去掉了使用sum的建议呢？

File: python/oneflow/nn/graph/graph.py (hunk 1 above)
Comment (Ldpe2G) [reply python/oneflow/nn/graph/graph.py:163]: 好的

File: python/oneflow/nn/graph/graph.py (hunk 2 above)
Comment (Ldpe2G) [reply python/oneflow/nn/graph/graph.py:223]: 这里目的是告诉用户需要 loss 需要是 scalar tensor ，所以给一个例子 mean 就行，而且一般也是用 mean ，很少用 sum，所以就去掉了

Comment (strint) [reply python/oneflow/nn/graph/graph.py:223]: 了解了，这里可以改成这样
```
So you may call methods such as ``Tensor.mean()`` to make the loss tensor a scalar tensor.
```

**Review by strint — APPROVED** [review]

**Comment by doombeaker** [comment]
保险起见，麻烦贴个文档最终的截图吧

**Review by doombeaker — APPROVED** [review]
